api/api_storage.go

```diff
@@ -5,80 +5,12 @@ import (
 
 	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/go-sectorbuilder"
+	s2 "github.com/filecoin-project/go-storage-miner"
 )
 
-// alias because cbor-gen doesn't like non-alias types
-type SectorState = uint64
-
-const (
-	UndefinedSectorState SectorState = iota
-
-	// happy path
-	Empty
-	Packing // sector not in sealStore, and not on chain
-
-	Unsealed      // sealing / queued
-	PreCommitting // on chain pre-commit
-	WaitSeed      // waiting for seed
-	Committing
-	CommitWait // waiting for message to land on chain
-	FinalizeSector
-	Proving
-	_ // reserved
-	_
-	_
-
-	// recovery handling
-	// Reseal
-	_
-	_
-	_
-	_
-	_
-	_
-	_
-
-	// error modes
-	FailedUnrecoverable
-
-	SealFailed
-	PreCommitFailed
-	SealCommitFailed
-	CommitFailed
-	PackingFailed
-	_
-	_
-	_
-
-	Faulty        // sector is corrupted or gone for some reason
-	FaultReported // sector has been declared as a fault on chain
-	FaultedFinal  // fault declared on chain
-)
+type SectorState = s2.SectorState
```

Comment:
I'd just remove this, and update references in lotus code

Reply:
Sounds good.